Makefile

```diff
@@ -7,25 +7,29 @@ TMPDIR := $(shell mktemp  -u /tmp/fooXXXXXX)
 all: setup bundle
 
 # Create the web pages in bundle/
-bundle: bundle/js/bootstrap-italia.min.js bundle/out.js index.html spectral.yml
-	cp index.html spectral.yml bundle
+bundle: bundle/js/bootstrap-italia.min.js copy_public bundle/out.js index.html spectral.yml
 
-bundle/out.js: index.js package.json setup
-	npx browserify --outfile bundle/out.js --standalone api_oas_checker index.js
+bundle/out.js: public/js/bundle.js package.json setup
+	npx browserify --outfile bundle/out.js --standalone api_oas_checker public/js/bundle.js
 
 gh-pages: bundle rules
 	rm css js asset svg -fr
 	git clone . $(TMPDIR) -b gh-pages
-	cp bundle/index.html bundle/spectral.yml bundle/out.js $(TMPDIR)
-	git -C $(TMPDIR) add index.html out.js spectral.yml
+	cp -r bundle/* $(TMPDIR)
+	git -C $(TMPDIR) add *
```

Comment:
@sebbalex ok but this may add to gh-pages files that accidentally landed there :) It's not an issue for now.